src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -308,7 +308,8 @@ protected function getCategoryListRepresentation(Request $request, $parentKey =
         $listBuilder->addSelectField($fieldDescriptors['locale']);
         $listBuilder->addSelectField($fieldDescriptors['defaultLocale']);
 
-        $listBuilder->addGroupBy($fieldDescriptors['id']);
+        // I think this "group by" is not required
+        //$listBuilder->addGroupBy($fieldDescriptors['id']);
```

Comment:
That's also what has been done in https://github.com/sulu/sulu/pull/1722/files#diff-6bf1cafae0624d44bf7a78e1896f12b4L311, so I guess that would be ok. But please remove it completely, not only comment it out. And add a line to the CHANGELOG.md 😃